scripts/nym-node-setup/auto-bond/auto_bond_all.py

```
@@ -0,0 +1,124 @@
+REPO_ROOT = Path(__file__).resolve().parents[3]  # scripts/nym-node-setup/autobond → up 3
+NYM_CLI   = REPO_ROOT / "target" / "release" / "nym-cli"
+ANSIBLE_PB = REPO_ROOT / "ansible" / "auto-bond.yml"
+
+
+DRY_RUN = "--dry-run" in sys.argv
+
+def run(cmd: list, capture=True) -> subprocess.CompletedProcess:
+    print(f"  $ {' '.join(str(c) for c in cmd)}")
+    if DRY_RUN:
+        return subprocess.CompletedProcess(cmd, 0, stdout='{"dry_run": true}', stderr="")
+    return subprocess.run(cmd, capture_output=capture, text=True, check=True)
```

Ruff (0.15.15)

[error] `subprocess` call: check for execution of untrusted input

(S603)